meta-mender-raspberrypi/README.md

```diff
@@ -0,0 +1,61 @@
+# meta-mender-raspberrypi
+
+Mender integration layer for Raspberry Pi family of boards. Note that this layer
+only contains templates, and the actually integration patches live in
```

Comment:
Minor nit - I think this should be and the *actual* integration pactches insetad of and the *actually* integration packages.

Reply:
Thanks fixed.